autoload/black.vim

```diff
@@ -56,6 +56,16 @@ def _get_virtualenv_site_packages(venv_path, pyver):
   return venv_path / 'lib' / f'python{pyver[0]}.{pyver[1]}' / 'site-packages'
 
 def _initialize_black_env(upgrade=False):
+  if vim.eval("g:black_use_virtualenv ? 'true' : 'false'") == "false":
```

Comment:
```suggestion
  if not vim.eval("g:black_use_virtualenv"):
```
Is this idiomatic? Can we not check the original value like this or something? Sorry, not super familiar 🤔

Reply:
I'm not familiar either, and I'm open to better ways. I thought pretty carefully about this and tried to write it in a way that has no possibility of unexpected behavior.

Vim's truthy/falsey handling is rather different from python's.
https://vi.stackexchange.com/questions/36270/what-values-are-truthy-and-falsey
https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/

For the values set in the vimrc file, since the vimrc file is in vimscript, we want to be able to respect vim's handing of truthiness. The eventual handling within python code is abstracted away from the user (the user would have no way of knowing this without examining the plugin source code).

`vim.eval()` operates on vimscript, so vim's truthiness comes into play. This is what I wanted, but unfortunately, `vim.eval()`, as documented, doesn't return ever a boolean. From `:help python-eval` in vim:
```
vim.eval(str)                                           python-eval
        Evaluates the expression str using the vim internal expression
        evaluator (see expression).  Returns the expression result as:
        - a string if the Vim expression evaluates to a string or number
        - a list if the Vim expression evaluates to a Vim list
        - a dictionary if the Vim expression evaluates to a Vim dictionary
```

To get around that limitation, I used the ternary operator.

1. The ternary operator in vimscript converts arbitrary truthy/falsey values to known strings.
2. `vim.eval()` returns one of these two known strings.
3. Python does string comparison to make its decision.

Reply:
Alright, thank you for the detailed explanation!

Reply:
You're welcome. I could have used any string values, but `"true"` and `"false"` looked the most clear to me.

I couldn't have done this instead:
```
if vim.eval("g:black_use_virtualenv ? '1' : '0'"):
```
...because python considers the string `"0"` to be true.


This does work:
```
if vim.eval("g:black_use_virtualenv ? '' : '1'"):
```
...but I think the original version is more clear to the reader.